### PR TITLE
doc: Clarify `MultiMesh.set_instance_color` re: white albedo color

### PR DESCRIPTION
**Pull request by CristianoSimao**
If the user not set the albedo color to pure white, then the multiplication with a diferent value than 1 will produce color totally wrong.
The pure white color r 255, g 255, b 255 internaly in Godot is 1 then the multiplication work ok.

### CODE REVIEW

File: doc/classes/MultiMesh.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
 			<argument index="1" name="color" type="Color" />
 			<description>
 				Sets the color of a specific instance by [i]multiplying[/i] the mesh's existing vertex colors.
-				For the color to take effect, ensure that [member use_colors] is [code]true[/code] on the [MultiMesh] and [member BaseMaterial3D.vertex_color_use_as_albedo] is [code]true[/code] on the material.
+				For the color to take effect, ensure that [member use_colors] is [code]true[/code] on the [MultiMesh], [member BaseMaterial3D.vertex_color_use_as_albedo] is [code]true[/code] on the material and albedo color of the material is set to pure white R: 255, G: 255, B: 255
```
Comment (Calinou) [inline doc/classes/MultiMesh.xml:57]: I would reword this as it's not necessarily incorrect to have a non-white albedo color:

```suggestion
				For the color to take effect, ensure that [member use_colors] is [code]true[/code] on the [MultiMesh], [member BaseMaterial3D.vertex_color_use_as_albedo] is [code]true[/code] on the material. If the color doesn't look as expected, make sure the material's albedo color is set to pure white ([code]Color(1, 1, 1)[/code]).
```

**Comment by akien-mga** [comment]
Pushed an update with Calinou's edit and a reworded commit message for clarity.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Comment by akien-mga** [comment]
Cherry-picked for 3.6.

**Comment by akien-mga** [comment]
Cherry-picked for 3.5.1.
